src/main/java/formflow/library/ScreenController.java

```diff
@@ -338,6 +338,8 @@ private ModelAndView handlePost(
       submission.setInputData(formSubmission.getFormData());
     }
 
+    ShortCodeConfig.Config config = shortCodeConfig.getConfig(flow);
```

Comment:
Should we add a check for if this is null with a helpful log to the user? 

Maybe something like `No shortcode configuration found for flow {}, no shortcode will be generated.`

Reply:
Yeah, good call here!